build.gradle

```diff
@@ -219,7 +226,7 @@ project(':iceberg-bundled-guava') {
   }
 
   shadowJar {
-    classifier null
+    archiveClassifier.set(null)
```

Comment:
nit: should `archiveClassifier = 'empty'` be changed to `archiveClassifier.set('empty')` in L251?

Reply:
same for https://github.com/apache/iceberg/blob/2d5a7091c16c167428e0cd6d932d8b59b4677b0f/deploy.gradle#L42

Reply:
updated those